test/unit/model/virtualbox-test.js

```diff
@@ -240,10 +240,23 @@ describe('Virtualbox installer', function() {
       }
     });
 
+    it('should run virtualbox installer executable to extract msi installer into temp folder', function(done) {
+      let stub = sandbox.stub(child_process, 'execFile').yields(null, "Files were extracted to temp", "");
+      try {
+        installer.install(fakeProgress, success, failure);
+        done();
+      } catch (error) {
+        expect.fail();
+      }
+      return helper.execFile().then(() => {
```

Comment:
if you stub everything properly all the work should be done inside main thread an you can just check with expect() here, there is no need to return promise instance from it.